src/Test/L0/Plugin/LogPluginHostL0.cs

```diff
@@ -0,0 +1,610 @@
+using Microsoft.TeamFoundation.DistributedTask.WebApi;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Xunit;
+using System.Threading;
+using System.Collections.ObjectModel;
+using Pipelines = Microsoft.TeamFoundation.DistributedTask.Pipelines;
+using Agent.Sdk;
+
+namespace Microsoft.VisualStudio.Services.Agent.Tests.LogPluginHost
+{
+    public sealed class LogPluginHostL0
+    {
+        // private IExecutionContext _jobEc;
```

Comment:
Should this commented-out code be removed? Or are there plans to add more L0 tests that depend on this code?

Reply:
deleted..